processor/memorylimiterprocessor/go.mod

```diff
@@ -4,6 +4,7 @@ go 1.18
 
 require (
 	github.com/stretchr/testify v1.8.1
+	github.com/tilinna/clock v1.1.0
```

Comment:
Let's avoid non standard dependencies (with super limited usage/contributions) like github.com/tilinna/clock

Reply:
Would github.com/benbjohnson/clock be an reasonable alternative?

The reason why I've used this original package is that it includes support for:
- Setting/Getting clock via context (Useful for not breaking interface abstractions)
- Includes a check to see how many active timers / tickers.

We've used this rather heavily in [gostatsd](https://github.com/atlassian/gostatsd) and some other internal projects, so I can at least speak to its production readiness.